[DOC] Improve project description and README (#39)
process-analytics/bpmn-visualization-R: README.md

</div>  
<br>

> This R package is an interface to the [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) library (TypeScript). \
> It enables to visualize the execution data of the processes on BPMN diagrams, using overlays, style customization and interactions.

Reviewer: Using something closer to the repository description

```suggestion
A R package for visualizing process execution data on BPMN diagrams, using overlays, style customization and interactions.

It is an interface to the [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) TypeScript library binds to R thanks to [htmlwidgets](http://www.htmlwidgets.org/).
```

Author: ```suggestion
This R package enables to visualize the execution data of the processes on BPMN diagrams, using overlays, style customization and interactions, with [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) library (TypeScript). \
Using JavaScript in R is handled by [htmlwidgets](http://www.htmlwidgets.org/) framework.
```

Reviewer: Final proposal agreed with @csouchet 

```suggestion
This project is an R package for visualizing process execution data on BPMN diagrams, using overlays, style customization and interactions.

It is made possible by [htmlwidgets](http://www.htmlwidgets.org/), which provides an easy-to-use framework for bringing together R and the [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) TypeScript library.
```

> This R package is an interface to the [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) library (TypeScript). \
> It enables to visualize the execution data of the processes on BPMN diagrams, using overlays, style customization and interactions.

Author: ```suggestion
This R package enables to visualize the execution data of the processes on BPMN diagrams, using overlays, style customization and interactions, with [BPMN Visualization](https://github.com/process-analytics/bpmn-visualization-js) library (TypeScript).
```

bpmn_file <- "<?xml version="1.0" encoding="UTF-8"?> ...xml_content"
```

> 💡 The package provides some [BPMN examples](./inst/examples). \

Reviewer: nice ❤️
